docs/source/cheat_sheet.rst

```diff
@@ -149,6 +149,15 @@ When you're puzzled or when things are complicated
    reveal_type(c)  # -> error: Revealed type is 'builtins.list[builtins.str]'
    print(c)  # -> [4] the object is not cast
 
+   # if you want dynamic attributes on your class, have it override __setattr__ in a stub
```

Comment:
Looks like you put it in the py3 but not the py2 cheat sheet. It should be the same in both.